products/monitoring/api.yaml

```diff
@@ -855,4 +857,189 @@ objects:
         without removing the channel from all alerting policies that reference the channel.
         This is a more convenient approach when the change is temporary and you want
         to receive notifications from the same set of alerting policies on the channel
-        at some point in the future.
\ No newline at end of file
+        at some point in the future.
+  
+  
+  - !ruby/object:Api::Resource
+    name: UptimeCheckConfig
+    update_verb: :PATCH
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation':
+          'https://cloud.google.com/monitoring/api/v3/'
+      api: 'https://cloud.google.com/monitoring/api/ref_v3/rest/v3/projects.uptimeCheckConfigs'
+    base_url: projects/{{project}}/uptimeCheckConfigs
+    self_link: "{{name}}"
+    description: This message configures which resources and services to monitor for
+      availability.
+    properties:
+    - !ruby/object:Api::Type::String
+      name: name
+      output: true
+      description: A unique resource name for this UptimeCheckConfig. The format is:projects/[PROJECT_ID]/uptimeCheckConfigs/[UPTIME_CHECK_ID].
+        on create the resource name is assigned by the server and included in the response.
```

Comment:
Is this sentence needed? I think it's implied by the fact that it's an output field.